monitoring/uss_qualifier/scd/data_interfaces.py

```diff
@@ -3,52 +3,59 @@
 from monitoring.monitorlib.scd_automated_testing.scd_injection_api import InjectFlightRequest
 from monitoring.uss_qualifier.common_data_definitions import Severity
 
+
 class KnownIssueFields(ImplicitDict):
-    ''' A class to hold a message that the test executor can provide to the USS in cases when the USS provides a response that is not same as the expected result. '''    
+    """Information, which can be defined at the time of test design, about a problem detected by an automated test when a USS provides a response that is not same as the expected result"""
     test_code: str
-    '''Code corresponding to check generating this issue'''
+    """Code corresponding to check generating this issue"""
 
     relevant_requirements: List[str] = []
-    '''Requirements that this issue relates to'''
+    """Requirements that this issue relates to"""
 
     severity: Severity
-    '''How severe the issue is'''
+    """How severe the issue is"""
 
     subject: Optional[str]
-    ''' Identifier of the subject of this issue, if applicable. This may be a UAS serial number, or any field of other object central to the issue. '''
+    """Identifier of the subject of this issue, if applicable. This may be a UAS serial number, or any field or other object central to the issue."""
 
     summary: str
-    '''Human-readable summary of the issue'''
+    """Human-readable summary of the issue"""
 
     details: str
-    '''Human-readable description of the issue'''
+    """Human-readable description of the issue"""
+
 
 class KnownResponses(ImplicitDict):
-    ''' A class to evaluate results / response to an injection of test flight data (TestFlightRequest). This data structure holds information about the "result" response provided by the test interface of the USS under test and details of how long the executor should wait for a response. Per the SCD Testing API an acceptable result should be one of "Planned", "Rejected", "ConflictWithFlight" or "Failed". In the case where the USS under test provides a result that is not the same as expected result then the test driver can provide a message detailing why the expected result was populated the way it was. This should help the USS under test debug specific parts of their internal systems. '''
+    """Mapping of the flight injection attempt's USS response to test outcome"""
     acceptable_results: List[str] 
-    ''' Acceptable strings that the USS under test can report as the result of processing the test data. '''
+    """Acceptable InjectFlightResponse `result` values that will result in the flight injection attempt being considered successful if the USS under test reports one of them as the result of attempting to inject the requested flight."""
```

Comment:
Consider: Acceptable values in the `result` data field of InjectFlightResponse. The flight injection attempt will be considered successful if the USS under test reports one of these as the result of test data processing and / or  attempting to inject the flight.

Reply:
Mostly adopted, but after removing "test data processing and / or".  The injection endpoint is semantically an endpoint to trigger creation of a flight with the given characteristics.  The data provided has to be processed, but that is incidental to the goal of treating the InjectFlightRequest as an attempt by a user to create a flight with those specifications.  If a USS had a bespoke routine (one not involved in the user-created flight creation pipeline) to first process the incoming test data before forwarding it on to the subsystem that creates the flight, that would essentially be cheating the automated test since logic not involved in the production system would be responsible for the success of some of the automated tests.